changelog.md

```diff
@@ -1,4 +1,7 @@
 # Future
+- [Fixed] `findAndCountAll` generates wrong SQL query [#5445](https://github.com/sequelize/sequelize/issues/5445)
+- [Fixed] `count` methods pollute the options.includes [#4191](https://github.com/sequelize/sequelize/issues/4191)
```

Comment:
Yes its want broken because someone fixed .findAll method, But still there was no test. So its a verification only, it was already fixed